src/napari_blob_detection/_widget.py

```diff
@@ -15,30 +15,53 @@ class Dimensionality(Enum):
 
 class Algorithm(Enum):
     DOG = auto()
-    DOH = auto()
     LOG = auto()
 
 
 ALGORITHM_LOOKUP = {
     Algorithm.DOG: blob_dog,
-    Algorithm.DOH: blob_doh,
     Algorithm.LOG: blob_log,
 }
 
 
 def detect_blobs(
     image: "napari.layers.Image",
     min_sigma: float = 1,
-    max_sigma: float = 50,
-    threshold: float = 0.5,
+    max_sigma: float = 10,
+    threshold: float = 0.1,
     dimensionality: Dimensionality = Dimensionality.TWO_D,
     algorithm: Algorithm = Algorithm.DOG,
 ) -> "napari.types.LayerDataTuple":
+
+    """
+    
+    Parameters
+    ----------
+    image : napari.layers.Image
+        Image layer for blob detection. Can be a 2D, 3D, or higher dimensionality image.
+    min_sigma : float
+        The smallest blob size to detect.
+    max_sigma : float
+        The largest blob size to detect.
+    threshold : float
+        Reduce this to detect blobs with lower intensities.
+    dimensionality: 
+        Specify if the image is 2D(+t) or 3D(+t).
```

Comment:
I think of this as more like the dimensionality of the features that you want to detect. It would also be good to mention how we handle extra dimensions (i.e. we use the last two or three dimensions of the image as the feature dimensions, then iterate over the other ones).